user-interface/6001-SORD-submit_orders.md

```diff
@@ -0,0 +1,139 @@
+  - **should** see a link to deposit the required collateral <a name="6001-SORD-050" href="#6001-SORD-050">6001-SORD-050</a>
+
+- **must** select a side/direction e.g. long/short (note: some implementations may do this with two different submit buttons long/short rather than a toggle) <a name="6001-SORD-004" href="#6001-SORD-004">6001-SORD-004</a>
+
+- **must** be able to select the [order type](../protocol/0014-ORDT-order_types.md) that I wish to submit <a name="6001-SORD-005" href="#6001-SORD-005">6001-SORD-005</a>
+  - **must** see limit order <a name="6001-SORD-006" href="#6001-SORD-006">6001-SORD-006</a>
+  - **must** see market order <a name="6001-SORD-007" href="#6001-SORD-007">6001-SORD-007</a>
+  - **should** see pegged order <!-- <a name="6001-SORD-008" href="#6001-SORD-008">6001-SORD-008</a> -->
+  - **should** see liquidity provision <!-- <a name="6001-SORD-009" href="#6001-SORD-009">6001-SORD-009</a> -->
+
+## Order size
+
+...need to select a size, when selecting a size for my order, I...
+
+- **must** input an order [size](7001-DATA-data_display.md#size) (aka amount or contracts) <a name="6001-SORD-010" href="#6001-SORD-010">6001-SORD-010</a>
+  - **should** have the previous value pre-populated (last submitted or last changed) <a name="6001-SORD-012" href="#6001-SORD-012">6001-SORD-012</a>
+  - **should** be able to hit up/down on the keyboard to increase the size by the market's min-contract size <a name="6001-SORD-013" href="#6001-SORD-013">6001-SORD-013</a>
+  - TODO **would** like to be able to use use a leverage slider to determine a size based on how much leverage I wish to use (given general balance, price input/current price) <!-- <a name="6001-SORD-015" href="#6001-SORD-015">6001-SORD-015</a> -->
+- **must** be warned (pre-submit) if input has too many decimal places for the market's ["position" decimal places](7001-DATA-data_display.md#size) <a name="6001-SORD-016" href="#6001-SORD-016">6001-SORD-016</a> 
+
+... so I get the size of exposure (open volume that I want)
+
+## Price - Limit order
+
+... if wanting to place a limit on the price that I trade at, I...
+
+- **must** enter a [price](7001-DATA-data_display.md#quote-price). <a name="6001-SORD-017" href="#6001-SORD-017">6001-SORD-017</a> 
+- **must** see the price unit (as defined in market) <a name="6001-SORD-018" href="#6001-SORD-018">6001-SORD-018</a>
+  - if the input field is empty: with the input focused: **should** be able to hit up/down to populate the input with the current mark price (if there is one, 0 if not) <a name="6001-SORD-011" href="#6001-SORD-011">6001-SORD-011</a>
+  - **should** have the previous value pre-populated (last submitted or last changed) <a name="6001-SORD-014" href="#6001-SORD-014">6001-SORD-014</a>
+  - **should** be able to hit up/down on the keyboard to increase the price by the market's tick size (if set, or smallest increment) <a name="6001-SORD-051" href="#6001-SORD-051">6001-SORD-051</a>
+
```

Comment:
Would like to see an estimated VWAP for the order size AND estimate of how much of the order will trade, OR a note that the order is likely to be passive on the book.

Reply:
again I think this is covered in 6001-SORD-033 and the "See the potential consequences of an order before it is submit" section

Reply:
https://github.com/vegaprotocol/specs/blob/user-interface-SORD-only/user-interface/6001-SORD-submit_orders.md#see-the-potential-consequences-of-an-order-before-it-is-submit

Reply:
added more detail here: https://github.com/vegaprotocol/specs/pull/1181/commits/264d32760ac920b54087ed686afc3a66db6f8dac